Sources/Sentry/SentryNSArrayRingBuffer.h

```diff
@@ -0,0 +1,27 @@
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * A ring buffer backed by @c NSArray that can return an @c NSArray containing its elements in the
+ * order in which they were added.
+ */
+@interface SentryNSArrayRingBuffer<T> : NSObject
```

Comment:
`h`: This file should be under `Sources/Sentry/include/` now it's under `Sources/Sentry`.

Reply:
That's the folder for headers we publicly expose to SDK consumers, right? If so, I don't think this kind of thing should go in there. But, I'm actually going to remove the ring buffer due to https://github.com/getsentry/sentry-cocoa/pull/1910#discussion_r907827172